packages/compiler/test/checker/completion.ts

```diff
@@ -139,6 +139,6 @@ describe("cadl: completion", () => {
 
     ok(node?.kind === SyntaxKind.Identifier, "expected to find an identifier");
     ok(testHost.program.checker, "program should have a checker");
-    return testHost.program.checker.resolveCompletions(node);
+    return Array.from(testHost.program.checker.resolveCompletions(node).keys());
```

Comment:
I really need to make things so you can test the language server itself. I will work on that soon.